modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -210,7 +250,7 @@ After you update your code and your tests are green, you are ready to release a
 <dependency>
    <groupId>org.mule.sdk</groupId>
    <artifactId>mule-sdk-api</artifactId>
-   <version>0.7.0</version>
+   <version>0.8.1</version>
```

Comment:
Does 0.8.1 exist? I believe 0.8.0 is the latest

Reply:
Actually isn't it 0.10.1?
https://mvnrepository.com/artifact/org.mule.sdk/mule-sdk-api?repo=mulesoft-public

Reply:
seems about right